### DISCUSSION TITLE
Copilot: Accept Solution button in Visual Studio is cut off

### DISCUSSION THREAD

**Discussion by Keboo** [community/community#33538] [Copilot Conversations]
Accessibility issue for users that increase the default font size.

Reproduction steps:
1. Increase the default font size within visual studio. 
2. Open the GitHub Copilot window (default keyboard short cut Ctrl+Alt+Enter).

Observed: The "Accept Solution" is cut off
![image](https://user-images.githubusercontent.com/952248/191124292-4ab2c08e-7629-460f-8423-268f8f1a3a2d.png)

Expected: The button should be visible
![image](https://user-images.githubusercontent.com/952248/191124323-d06579ef-fd15-431b-a1c6-e604cc17137d.png)

Additional details:
It appears the the button has a fixed size of 100x22. This is in CopilotWindowControl.xaml within GitHub.Copilot.Vsix.dll. Simply removing the hard coded assigned values, or setting to `double.NaN` will allow the buttons to expand to their content. Since they are already inside of a `StackPanel` the rest of the content will respond accordingly.

**Comment by iamcjwilliams** [comment]
Thank you! @Keboo  Our copilot team is investigating this issue
